plugins/usbdmx/WidgetFactory.h

```diff
@@ -244,7 +269,6 @@ bool BaseWidgetFactory<WidgetType>::AddWidget(WidgetObserver *observer,
 template <typename WidgetType>
 void BaseWidgetFactory<WidgetType>::DeviceRemoved(WidgetObserver *observer,
                                                   libusb_device *usb_device) {
-  OLA_INFO << "Removing device " << usb_device;
```

Comment:
Ah. Do we log it somewhere else in the stack then? It seems worth having for debug.